apps/meteor/client/views/omnichannel/contactInfo/tabs/ContactInfoDetails/ContactInfoDetailsEntry.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, Icon, IconButton } from '@rocket.chat/fuselage';
+import { useTranslation } from '@rocket.chat/ui-contexts';
+import React from 'react';
+
+import { useIsCallReady } from '../../../../../contexts/CallContext';
+import useClipboardWithToast from '../../../../../hooks/useClipboardWithToast';
+import { parseOutboundPhoneNumber } from '../../../../../lib/voip/parseOutboundPhoneNumber';
+import ContactInfoCallButton from './ContactInfoCallButton';
+
+type ContactInfoDetailsEntryProps = {
+	type: 'phone' | 'email';
+	label: string;
+	value: string;
+};
+
+const ContactInfoDetailsEntry = ({ type, value }: ContactInfoDetailsEntryProps) => {
+	const t = useTranslation();
+	const { copy } = useClipboardWithToast(value);
+
+	const isCallReady = useIsCallReady();
+
+	return (
+		<Box display='flex' alignItems='center'>
+			<Icon size='x18' name={type === 'phone' ? 'phone' : 'mail'} />
+			<Box withTruncatedText display='flex' flexGrow={1} alignItems='center' justifyContent='space-between'>
+				<Box withTruncatedText data-qa-id={type === 'phone' ? 'contactInfo-phone' : 'contactInfo-email'} mi={4} fontScale='p2'>
```

Comment:
i dont like components that receive props to behave differently based on the prop. its not generic 

pass the formatted vale/icon etc

also data-qa? are we missing some accessibility info here?